pkg/controller/controller.go

```diff
@@ -529,12 +523,15 @@ func (c *Controller) syncHandler(key string) error {
 		}
 
 		// Get analytics worker deployment name using hardcoded value
-		workerdeployment, err := c.deploymentsLister.Deployments(apimanager.Namespace).Get(workerDeploymentName)
+
+		// workerdeployment, err := c.deploymentsLister.Deployments(apimanager.Namespace).Get(workerDeploymentName)
```

Comment:
Shall we remove these commented lines?

Reply:
Commented lines do not exist in the master branch

Reply:
Seems like the line numbers are different in master https://github.com/wso2/k8s-wso2am-operator/blob/master/pkg/controller/controller.go#L517

Reply:
fixed via https://github.com/wso2/k8s-wso2am-operator/pull/90/commits/a576020688d42ebe8201635fa8d9689cffe0437e